### DISCUSSION TITLE
Filtrar OSs pelo técnico/reponsável

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1780] [Q&A]
Saudações galera!!!

Essa url é para pesquisar o cliente teste com estado de Aberta: index.php/os/gerenciar?pesquisa=teste&status=Aberto&data=&data2=

Como seria uma URL para filtrar o técnico?

Valeuuuu!!!!!

**Comment by Pr3d4dor** [comment]
@bernarcom Essa funcionalidade ainda não existe.

**Reply by SuperPorte** [reply]
Como criaria uma url para essa pesquisa? Ou não é só a URL que faz isso?
Não manjo nada de programação.

**Reply by Pr3d4dor** [reply]
Teria que ser implementado no sistema mesmo essa busca.
